package.json

```diff
@@ -84,6 +84,8 @@
     "file-loader": "^6.2.0",
     "husky": "^3.0.0",
     "lint-staged": "^9.2.0",
+    "mocha": "^8.3.2",
```

Comment:
better keep original versions, that's the likeliest reason behind broken tests

Reply:
Yeah, actually it was, reverted